scripts/battlefields/Apollyon/central_apollyon.lua

```diff
@@ -26,7 +26,7 @@ local content = Limbus:new({
 })
 
 function content:isValidEntry(player, npc)
-    return self.entryNpc == '_12i' or self.entryNpc == '_127'
+    return npc and (npc:getName() == '_12i' or npc:getName() == '_127')
```

Comment:
```
for _, npcName in self.entryNpcs do
    if npc:getName == npcName then
        return true
    end

    return false
```

Reply:
entryNpcs is a table, entryNpc on construct is now converted to a table, so this was a miss on my part

Reply:
Also set entryNpcs in the constructor to { '_12i, '_127' }

Reply:
Most likely now that I think of it, you won't even need the entry function in this BCNM

Reply:
Got it, so for both we just need to remove the isvalidentry override and defined entryNpcs in their constructors?

Reply:
I believe so.  Sorry for the mixed info as I was thinking through things, but yeah, that should work on its own

Reply:
Yep, simpler logic and it works.

Really love the new battlefield content structure. Easy to module changes, cleaner format, more things in one place